c31-pryhaven/nutshell2
# Av username appended

# Description

When the user posts a message, their username will be appended to the message.

Applies to issue # (issue number)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

```git fetch --all```
``git checkout av-username-appended```
navigate to /messages.
see that all users have a username.


# Checklist:
